src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class NoticeOfDispute
+    {
+        public string? TicketNumber { get; set; }
+        public string? ProvincialCourtHearingLocation { get; set; }
+        public DateTime? IssuedDate { get; set; }
+        public string? Surname { get; set; }
+        public string? GivenNames { get; set; }
+        public string? Address { get; set; }
+        public string? City { get; set; }
+        public string? Province { get; set; }
+        public string? PostalCode { get; set; }
+        public string? HomePhoneNumber { get; set; }
+        public string? WorkPhoneNumber { get; set; }
+        public string? EmailAddress { get; set; }
+        public IList<DisputedCount> DisputedCounts { get; set; } = new List<DisputedCount>();
```

Comment:
There is not need to declare the property is IList<T> when it is created as List<T>